aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1504,12 +1505,14 @@ impl VMExecutor for AptosVM {
         );
 
         let count = transactions.len();
+        let index_mapping = IndexMapping::new_unsharded(transactions.len());
```

Comment:
can we derive this from ExecutableTransaction so we don't need to pass extra param?

Reply:
possible, but we still need to augment `ExecutableTransactions` to have the block size.

Reply:
+1 to @zekun000's comment, its cleaner to have the block size being carried in the `ExecutableTransactions` itself.

Reply:
fixed